.github/workflows/playwright.yml

```diff
@@ -0,0 +1,44 @@
+name: Playwright Tests
+on:
+  push:
+    branches: [pathfinder]
+  pull_request:
+    branches: [pathfinder]
+jobs:
+  test:
+    timeout-minutes: 60
+    runs-on: self-hosted
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: pnpm/action-setup@v2
+        with:
+          version: 'latest'
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
+          registry-url: https://npm.pkg.github.com
+          scope: '@toeverything'
+          cache: 'pnpm'
+
+      - run: node scripts/module-resolve/ci.js
+
+      - name: Install dependencies
+        run: pnpm install --no-frozen-lockfile
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_GITHUB_AUTH_TOKEN }}
+
+      - name: Install Playwright Browsers
+        run: npx playwright install --with-deps
+
+      - name: Run Playwright tests
+        run: pnpm dlx playwright test
+
+      - uses: actions/upload-artifact@v3
+        if: always()
+        with:
+          name: playwright-report
+          path: playwright-report/
+          retention-days: 30
```

Comment:
There also have a repo setting, I suggest removing this setting to avoid confusion.

<img width="870" alt="Screenshot 2022-11-10 at 8 44 10 PM" src="https://user-images.githubusercontent.com/18554747/201094783-aa3e8c08-7d17-4e33-9d39-4305bd0e6414.png">